packages/svelte-query-persist-client/tests/PersistQueryClientProvider.svelte.test.ts

```diff
@@ -76,15 +74,14 @@ describe('PersistQueryClientProvider', () => {
         queryClient,
         persistOptions: { persister },
         key,
-        states: statesStore,
+        states,
       },
     })
 
     await waitFor(() => rendered.getByText('fetchStatus: idle'))
     await waitFor(() => rendered.getByText('hydrated'))
     await waitFor(() => rendered.getByText('fetched'))
 
-    const states = get(statesStore)
     expect(states).toHaveLength(4)
```

Comment:
@zhihengGet I've tried fixing some of the tests, but this behaviour is really stumping me. Previously, `statesStore` could be passed as a prop into the component, and when it was updated within the component, it would update within the test file.

I'm struggling to reproduce this two-way binding with runes. I've tried using the `$bindable()` rune without luck. I would greatly appreciate your input here!

Reply:
@lachlancollins maybe this would help? I'm the "mental" author of a new `$state()` for props too: https://github.com/sveltejs/svelte/issues/12451#issuecomment-2228760302

Reply:
> @lachlancollins maybe this would help? I'm the "mental" author of a new `$state()` for props too: [sveltejs/svelte#12451 (comment)](https://github.com/sveltejs/svelte/issues/12451#issuecomment-2228760302)

I don't think it's quite the same, since this isn't passing props between components, but through a test renderer to a component.

I've opened an issue here to see if anyone can help: https://github.com/testing-library/svelte-testing-library/issues/394

Reply:
This has been solved!